src/Rings/mpoly-ideals.jl

```diff
@@ -2009,48 +2009,61 @@ function Base.show(io::IO, I::MPolyIdeal)
   end
 end
 
-
 _pluecker_sgn(a::Vector{Int}, b::Vector{Int}, t::Int)::Int =
  iseven(count(z -> z > t, a) + count(z -> z < t, b)) ? 1 : -1
 @doc raw"""
-    flag_pluecker_ideal(F::Field, dimensions::Vector{Int},n::Int)
+    flag_pluecker_ideal(F::Union{Field, MPolyRing}, dimensions::Vector{Int},n::Int; minimal::Bool=true)
 
 Returns generators of the defining ideal for the flag variety
 $\text{Fl}(\mathbb{F}, (d_1,\dots,d_k), n)$, where $(d_1,\dots,d_k)
 =$`dimensions` denotes the rank, with $d_j\leq n-1$.  That is, the vanishing
 set of this ideal corresponds to the space of $k$-step flags of linear
 subspaces $V_1\subset\dots\subset V_k$ in $\mathbb{F}^n$, where
 $\text{dim}(V_j) = d_{j}$.  You can obtain the generators for the
-\emph{complete flag variety}$ of $\mathbb{F}^{n}$ by taking `dimensions`
+$\emph{complete flag variety}$ of $\mathbb{F}^{n}$ by taking `dimensions`
 $=(1,\dots,n-1)$ and `n`$=n$.  We remark that evaluating for `F = QQ` yields
 the same set of generators as any field of characteristic $0$.
+Evaluating this function with the parameter `minimal = true` returns the reduced Gröbner basis for
+the flag Plücker ideal with respect to the reverse lexographical order. The first parameter can
```

Comment:
```suggestion
the flag Plücker ideal with respect to the degree reverse lexographical order. The first parameter can
```
At least that's what happens in the code.